index.html

```diff
@@ -881,6 +870,14 @@ <h2 id="state-transitions" class="informative">
             </td>
           </tr>
         </table>
+        <div class="note" id="state-transitions">
+          <p>
+            The internal slot <a>[[\state]]</a> transitions between its various
+            potential values as follows:
+          </p><img alt=
```

Comment:
yeah, the alt is redundant... don't want to repeat it. Sorry for not being clear.